single-line-diagram/single-line-diagram-core/src/main/java/com/powsybl/sld/builders/NetworkGraphBuilder.java

```diff
@@ -632,6 +667,27 @@ private void addLineEdges(Graph graph, List<Line> lines) {
         });
     }
 
+    private void addTieLineEdges(Graph graph, List<TieLine> tieLines) {
```

Comment:
Duplicate code here which can be easily extended, as `TieLine` and `Line` extends `Branch` which is enough to what's done inside. Hence you could add a `addBranchEdges(Graph graph, List<? extends Branch<?>> branches)` method